src/icesat2_tracks/config/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.0.0"
```

Comment:
I don't think we need a __version__ here – maybe a docstring for the subpackage
```suggestion
"""Configuration parameters for the icesat2_tracks package"""
```

Reply:
Again, ok I'll change it. I just thought the version comment was mandatory on all __init__.py files